src/intervals.js

```diff
@@ -0,0 +1,30 @@
+import "./App.css";
+
+const Intervals = () => {
+  return (
+    <>
+      <div className="padding-container" id="intervals-section">
+        <div className="container-flex">
+          <div className="title">Intervals</div>
```

Comment:
we shouldn't use divs for everything. Divs are more like sectional dividers. For content like such, a paragraph or heading would make more sense semantically.